libobs-opengl/gl-cocoa.m

```diff
@@ -228,6 +228,10 @@ void gl_update(gs_device_t *device)
 	NSOpenGLContext *parent = device->plat->context;
 	NSOpenGLContext *context = swap->wi->context;
 	dispatch_async(dispatch_get_main_queue(), ^() {
+		if (!swap || !swap->wi) {
+			return;
+		}
```

Comment:
How do we feel about curly braces around one liners again? I thought we wanted to avoid them but I'm not sure.

Reply:
Discussed this off-thread, we're safely in "let's be explicit" territory now. 😄